Sources/LinuxPlatform/Linux.swift

```diff
@@ -125,6 +137,26 @@ public struct Linux: Platform {
                 "tzdata",
                 "zlib1g-dev",
             ]
+        case "ubuntu2404":
```

Comment:
I think we're missing 2310 from this list

Reply:
I couldn't find a list of packages for ubuntu 23.10 from the Dockerfiles. If someone knows where the docker file is, I can add it here.

This is where I usually look for them: https://github.com/swiftlang/swift-docker/tree/main/6.0/ubuntu

Reply:
I manually pieced together the list by looking at the differences between ubuntu 22.04, and 24.04. Tested it out in docker and it looks to be working fine. There's only the swift 5.10.1 release available for that one though, so I wonder how useful and used this version of ubuntu will be.

Reply:
I have removed 23.10 from the list because we can't bring up infrastructure for a platform that is EOL.